README.md

```diff
@@ -1,125 +1,85 @@
 
-# About
+# lsif-util
 
-Scripts to help [LSIF](https://github.com/Microsoft/language-server-protocol/blob/master/indexFormat/specification.md) developers. Featuring:
+Scripts to help [LSIF](https://github.com/Microsoft/language-server-protocol/blob/master/indexFormat/specification.md) developers.
 
-* Validation
-* Graph visualization
-* Quick search
+## What's new
 
-# Getting Started
+* _Up to date!_ Support for [JSON Lines](http://jsonlines.org/) with the `--inputFormat` tag
+* _Cleaning up!_ Standard filtering for all tools (no more unnecessary searching)
 
-* `> git clone this repository`
-* `> cd lsif-util`
-* `> npm install`
-* `> npm run compile`
+## What's next
 
-# Validation
+* _We are moving!_ The lsif-util tools will soon migrate to [lsif-node](https://github.com/microsoft/lsif-node)
+* _Getting official!_ Global npm package is on the way
 
-`> node .\lib\validate.js [options]`
+## Usage
 
-| Option            | Default     | Description                                             |
-|-------------------|-------------|---------------------------------------------------------|
-| --inputPath or -p | ./lsif.json | Path to input file (JSON)                               |
+``` bash
+git clone https://github.com/jumattos/lsif-util.git
```

Comment:
To make it easier to read, do you think we could split usage into installation and usage? It took me a while to realize that the inputFormat/stdin were for the last line in code block

Reply:
Done! Soon the "installation" part will be much easier. 😊